Feature detection mechanism cuba-platform/cuba-rest-js#20

vyacheslav-pushkin opened the pull request.

vyacheslav-pushkin committed:
Feature detection mechanism cuba-platform/cuba-rest-js#20

web-devel commented on src/app/MainStore.ts:
```diff
@@ -92,8 +92,14 @@ export class MainStore {
-    this.cubaREST.setSessionLocale();
+    this.cubaREST.setSessionLocale().catch((reason) => {
+      if (reason === CubaApp.NOT_SUPPORTED_BY_API_VERSION) {
+        throw new Error('Relogin is required in order for bean validation messages to use correct locale. ' +
```
maybe just log it to console?

vyacheslav-pushkin committed:
Feature detection mechanism cuba-platform/cuba-rest-js#20

When setSessionLocale is not supported log a warning instead
of throwing an error
